src/reducers/computeRenderedRows.js

```diff
@@ -188,7 +188,7 @@ function calculateRenderedRowRange(state, scrollAnchor) {
 
     // Handle a case where the offset puts the first row fully offscreen
     // This can happen if availableHeight & maxAvailableHeight are different
-    const { storedHeights } = state;
+    const { storedHeights } = state.getInternal();
```

Comment:
`storedHeights` is no longer a direct part of the redux store state. Instead `getInternal` is used to access it. 

The rest of the code changes in the PR are similar: Accessing internal state like `storedHeights`, `rowBufferSet`, and `rowOffsetIntervalTree` are now done through the getter.